driverinference/main.py

```diff
@@ -275,31 +283,33 @@ def _create_inference_session(cls,
                         num_devices=num_devices)
 
         return model, resnet_model
-    
+
     @classmethod
     def _create_colocated_inference_session(cls,
```

Comment:
in this file -> add language to the infer sess name, so users dont have to look at the csv file to check if fortran or cpp